content/en/serverless/installation/nodejs.md

```diff
@@ -37,184 +37,139 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 
-1. Clone the Datadog CloudFormation macro repository in your local environment:
-
-    ```
-    git clone https://github.com/DataDog/datadog-cloudformation-macro.git
-    ```
-
-2. Install the macro and it's dependencies:
-
-    ```
-    yarn install # Yarn users
-    npm install  # NPM users
-    ```
-
-3. Run the build script:
-
-    ```
-    yarn build    # Yarn users
-    npm run build # NPM users
-    ```
+```sh
+aws cloudformation create-stack \
+  --stack-name datadog-serverless-macro \
+  --template-url https://datadog-cloudformation-template.s3.amazonaws.com/aws/serverless-macro/latest.yml \
+  --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM
+```
 
-Deploy the macro to the same region as your Lambda functions. You need to deploy the macro in every region where you are monitoring Lambda functions.
+The macro is now deployed and ready to use.
 
-4. You will need an S3 bucket to store the CloudFormation artifacts for the macro. If you don't have one already, you can create one with:
+### Instrument the Function
+
+Add the `DatadogServerless` transform and the `CfnMapping` to your `Stack` object in your AWS CDK app. See the sample code below in TypeScript (the usage in other language should be similar).
+
+```typescript
+import * as cdk from "@aws-cdk/core";
+
+class CdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+    this.addTransform("DatadogServerless");
+
+    new cdk.CfnMapping(this, "Datadog", {
+      mapping: {
+        Parameters: {
+          nodeLayerVersion: "<LAYER_VERSION>",
+          forwarderArn: "<FORWARDER_ARN>",
+          stackName: this.stackName,
+          service: "<SERVICE>",
+          env: "<ENV>",
+        },
+      },
+    });
+  }
+}
+```
 
-    ```bash
-      aws s3 mb s3://<bucket name>
-    ```
+Replace `<SERVICE>` and `<ENV>` with appropriate values, `<LAYER_VERSION>` with the desired version of Datadog Lambda layer (see the [latest releases][3]), and `<FORWARDER_ARN>` with Forwarder ARN (see the [Forwarder documentation][2]).
 
-5. Package the provided CloudFormation template (`macro_template.yml`). This includes a Lambda function and a CloudFormation macro resource. The provided template uses the AWS Serverless Application Model (SAM), so it must be transformed before deployment:
 
-    ```bash
-    aws cloudformation package \
-        --template-file macro_template.yml \
-        --s3-bucket <your bucket name here> \
-        --output-template-file packaged.template
-    ```
+[1]: https://github.com/DataDog/datadog-cloudformation-macro
+[2]: https://docs.datadoghq.com/serverless/forwarder/
+[3]: https://github.com/DataDog/datadog-lambda-js/releases
+{{% /tab %}}
+{{% tab "Datadog CLI" %}}
 
-6. Deploy the packaged CloudFormation template to a CloudFormation stack:
+<div class="alert alert-warning">This service is in public beta. If you have any feedback, contact <a href="/help">Datadog support</a>.</div>
```

Comment:
Isn't the Datadog CLI in private beta? Should we also just name the tab "CLI"?

Reply:
It was, post dash it's public now

Reply:
CLI seems too generic for me. Why do you think CLI is better?